README.md

```diff
@@ -25,10 +27,9 @@ Such that this can be used in URIs like:
 # JavaScript Library
 
 Using the JavaScript library we can construct queries
-using chained operator calls in JavaScript. We could execute the query above like this:
 
-    var Query = require("rql/query").Query;
-    var fooEq3Query = new Query().eq("foo",3);
+    import { RQLQuery } from '@swimlane/rql';
+	const rqlQuery = RQLQuery.parse(foo,3);
```

Comment:
Shouldn't the argument be in quotes/a string?

Reply:
fixed